.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -102,12 +102,12 @@ jobs:
           MAGMA_DEV_MEMORY_MB: 9216
         working-directory: lte/gateway
         # the image is directly concatenated to the registry, so we need the slash as delimiter
-        run: fab --show=debug --set DOCKER_REGISTRY=${{ inputs.registry }}/ integ_test_containerized:test_mode=${{ inputs.test_targets }}
+        run: fab -d integ-test-containerized --test-mode=${{ inputs.test_targets }} --docker-registry=${{ inputs.registry }}/
       - name: Get test results
         if: always()
         working-directory: lte/gateway
         run: |
-          fab get_test_summaries:dst_path="test-results",sudo_tests=False,dev_vm_name="magma_deb"
+          fab get-test-summaries --integration-tests --dst-path="test-results" --dev-vm-name="magma_deb"
```

Comment:
NIT `--dev-vm-name="magma_deb"` is not used because the flag `sudo_tests` is `False` - this was unnecessary before this change. `get-test-summaries` should be refactored anyway. The flags `sudo_tests` and `integration_tests` are at least confusing. We should do this in a follow-up.